promotion/datastore_test.go

```diff
@@ -739,3 +737,93 @@ func (suite *PostgresTestSuite) TestInsertClobberedClaims() {
 	err = pg.DB.Select(&allCreds2, `select * from clobbered_claims;`)
 	suite.Require().Equal(allCreds1, allCreds2, "creds should not be inserted more than once")
 }
+
+func (suite *PostgresTestSuite) TestDrainClaim() {
+	pg, err := NewPostgres("", false)
+	suite.Require().NoError(err)
+
+	publicKey := "hBrtClwIppLmu/qZ8EhGM1TQZUwDUosbOrVu3jMwryY="
+	blindedCreds := jsonutils.JSONStringArray([]string{"hBrtClwIppLmu/qZ8EhGM1TQZUwDUosbOrVu3jMwryY="})
```

Comment:
Shouldn't this be `publicKey` instead of the same string from above?

Reply:
technically it should be a distinct value in each - but here since we're mocking the calls I just reused it. so it may be more confusing to reference `publicKey`